src/main/resources/com/adobe/epubcheck/util/messages.properties

```diff
@@ -20,7 +20,7 @@ argument_needed=At least one argument expected
 version_argument_expected=Version number omitted from the -version argument.
 mode_argument_expected=Type omitted from the -mode argument.
 profile_argument_expected=Profile name omitted after the -profile argument.
-profile_unknown=Unknown profile '%1$s', using default validation profile.
+profile_unknown=Unknown profile '%1$s', usingÂ default validation profile.
```

Comment:
there seems to be a unicode character glitch - can you have a look @matthew-macgregor ?

Reply:
@rdeltour fixed this with commit 91ccd2f62fa944b44644be62aac098b06accccad